atd-vze/src/views/Crashes/Crash.js

```diff
@@ -134,6 +136,19 @@ function Crash(props) {
           }`}</h2>
         </Col>
       </Row>
+      {(isPrivateDrive !== false || isInAustinFullPurpose !== true) && (
+        <Row>
+          <Col>
+            <Alert color="warning">
```

Comment:
would you add the `fa-exclamation-triangle` icon to the left of this alert's text?